### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by DhuruvSingh**
### Mobile Menu

- Added Mobile menu with HTML, CSS  & JavaScript
- Added When the user clicks (or taps) the hamburger button, the mobile menu appears.
- Added When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- Added When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- Added When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

> Kindly Review and give your valuable feedback

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,13 +18,17 @@
...
-            <ul class="desktop-menu">
-                <li><a href="#main-sec">Portfolio</a></li>
-                <li><a href="#about-me">About</a></li>
-                <lI><a href="#contact-form">Contact</a></lI>
+            <!-- <div class="menu">
+            </div> -->
+            <ul class="menu nav-icon desktop-menu">
+                <li><a class="menuItem nav-menu" href="#main-sec">Portfolio</a></li>
+                <li><a class="menuItem nav-menu" href="#about-me">About</a></li>
+                <lI><a class="menuItem nav-menu" href="#contact-form">Contact</a></lI>
```
Comment (vickymarz) [inline index.html:26]: - [ ] You have done well implementing the functionality to toggle the mobile menu 📱 . I noticed a vertical scroll bar on the nav lit of the desktop site. Please fix it 👍 

image

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @DhuruvSingh , 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅  
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (DhuruvSingh) [reply index.html:26]: done

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @DhuruvSingh  and @vickymarz  🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- No linter errors ✅


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions
- [OPTIONAL ] I liked your project 💯 and I'd like to give you a small tip I suggest adding `scroll-behavior: smooth` (css-property) to the `<html>` element to enable smooth scrolling for the whole page [Link](https://www.w3schools.com/howto/howto_css_smooth_scroll.asp#section2) 😉👌

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DhuruvSingh** [comment]
Thank you so much for your valuable feedback @Bondok6 & @vickymarz

**Pull request merged**